cicecore/cicedynB/general/ice_init.F90

```diff
@@ -1141,198 +1141,189 @@ fpfunc_andacc = 1      ! fixed point function for Anderson acceleration: 1: g(x)
+               write(nu_diag,1030) ' yield_curve      = ', trim(yield_curve)
                if (trim(yield_curve) == 'ellipse') &
-               write(nu_diag,1007) ' e_ratio          = ', e_ratio, ' aspect ratio of ellipse'
+               write(nu_diag,1002) ' e_ratio          = ', e_ratio, ' : aspect ratio of ellipse'
             endif
 
             if (trim(coriolis) == 'latitude') then
-               tmpstr2 = ': latitude-dependent Coriolis parameter'
+               tmpstr2 = ' : latitude-dependent Coriolis parameter'
             elseif (trim(coriolis) == 'contant') then
-               tmpstr2 = ' = 1.46e-4/s'
+               tmpstr2 = ' : = 1.46e-4/s'
             elseif (trim(coriolis) == 'zero') then
-               tmpstr2 = ' = 0.0'
+               tmpstr2 = ' : = 0.0'
             else
                tmpstr2 = ': unknown value'
             endif
-            write(nu_diag,*) 'coriolis         = ',trim(coriolis),trim(tmpstr2)
+            write(nu_diag,1030) ' coriolis         = ',trim(coriolis),trim(tmpstr2)
 
             if (trim(ssh_stress) == 'geostrophic') then
-               tmpstr2 = ': from ocean velocity'
+               tmpstr2 = ' : from ocean velocity'
             elseif (trim(ssh_stress) == 'coupled') then
-               tmpstr2 = ': from coupled sea surface height gradients'
+               tmpstr2 = ' : from coupled sea surface height gradients'
             else
-               tmpstr2 = ': unknown value'
+               tmpstr2 = ' : unknown value'
             endif
-            write(nu_diag,*) 'ssh_stress       = ',trim(ssh_stress),trim(tmpstr2)
+            write(nu_diag,1030) ' ssh_stress       = ',trim(ssh_stress),trim(tmpstr2)
 
             if (ktransport <= 0) then
                advection = 'none'
             endif
             if (trim(advection) == 'remap') then
-               tmpstr2 = ': linear remapping advection'
+               tmpstr2 = ' : linear remapping advection'
             elseif (trim(advection) == 'upwind') then
-               tmpstr2 = ': donor cell (upwind) advection'
+               tmpstr2 = ' : donor cell (upwind) advection'
             elseif (trim(advection) == 'none') then
-               tmpstr2 = ': advection disabled by ktransport namelist'
+               tmpstr2 = ' : advection disabled by ktransport namelist'
             else
-               tmpstr2 = ': unknown value'
+               tmpstr2 = ' : unknown value'
             endif
-            write(nu_diag,*) 'advection        = ', trim(advection),trim(tmpstr2)
+            write(nu_diag,1030) ' advection        = ', trim(advection),trim(tmpstr2)
 
             if (basalstress) then
-               tmpstr2 = ' use basal stress parameterization for landfast ice'
+               tmpstr2 = ' : use basal stress parameterization for landfast ice'
             else
-               tmpstr2 = ' basal stress not used for landfast ice'
+               tmpstr2 = ' : basal stress not used for landfast ice'
```

Comment:
This description is a little bit misleading, in the sense that it could be read to mean that the code has other means of simulating landfast ice, besides the basal stress parameterization, which is not the case (Ktens can help but it's valid on its own)...

so maybe just "no basal stress parameterization" ? 

@JFLemieux73 what do you think ?

Reply:
I updated it as suggested.